packages/account-v2/src/constants/manualFormConstants.tsx

```diff
@@ -9,18 +9,18 @@ import ProofOfAgeIcon from '../assets/manual-upload/proof-of-age.svg';
 export type TManualDocumentTypes = typeof MANUAL_DOCUMENT_TYPES[keyof typeof MANUAL_DOCUMENT_TYPES];
 
 export const MANUAL_DOCUMENT_TYPES = Object.freeze({
-    DRIVING_LICENCE: 'driving_licence',
-    NATIONAL_IDENTITY_CARD: 'national_identity_card',
-    NIMC_SLIP: 'nimc_slip',
-    PASSPORT: 'passport',
+    driving_license: 'driving_license',
```

Comment:
The value returned by BE is `driving_licence` https://api.deriv.com/api-explorer#residence_list. Lets keep it that way